domains/antarixa.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "AdiAlamSami",
+    "email": "mrxnexsus@gmail.com"
+  },
+  "records": {
+    "CNAME": "https://kidii-porto.vercel.app/"
```

CNAME doesnt allow trailing slashes (/)
```suggestion
    "CNAME": "kidii-porto.vercel.app"
```